src/Compilers/CSharp/Portable/Emitter/Model/PEModuleBuilder.cs

```diff
@@ -389,6 +389,12 @@ private static void GetDocumentsForMethodsAndNestedTypes(PooledHashSet<Cci.Debug
 
                                     case SymbolKind.Property:
                                         AddSymbolLocation(result, member);
+                                        FieldSymbol fieldKeywordBackingField = (member as SourcePropertySymbolBase)?.FieldKeywordBackingField;
+                                        if (fieldKeywordBackingField is not null)
+                                        {
+                                            AddSymbolLocation(result, fieldKeywordBackingField);
```

Comment:
>AddSymbolLocation(result, fieldKeywordBackingField); [](http://example.com/codeflow?start=44&length=52)

Is this change backed by a test?

Reply:
Yes. `TestWinMdExpData_SemiAutoProperty`, but I don't exactly understand what the test is doing or what behavior can be seen from end-user perspective.